lms/services/vitalsource/_client.py

```diff
@@ -141,6 +144,9 @@ def get_user_credentials(self, user_reference: str) -> dict:
         if credentials := result["credentials"].get("credential"):
             return self._to_camel_case(self._pick_first(credentials))
 
+        LOG.exception(
+            "VitalSource user not found: %s", result["credentials"].get("error")
```

Comment:
We  could just log `credentials` because the previous `if` checks that credentials->credentials doesn't exist. 

Logging just the `error` just to be safe of any refactor that misses that.